docs/Contributing/API-for-contributors.md

```diff
@@ -617,9 +623,9 @@ Content-Type: application/octet-stream
 
 `Status: 200`
 
-### Upload ABM Token
+### Add ABM token
 
-`POST /api/v1/fleet/mdm/apple/abm_token`
+`POST /api/v1/fleet/abm_tokens`
```

Comment:
### Dev note

We can break this endpoint and others because they're in the contributor docs.